### PR TITLE
11 create manager and managerreview

### LINKED ISSUE #11: Create Manager and ManagerReview
Create entity Manager and ManagerReview

### PR DESCRIPTION
**Pull request by FirdavsAX**
Added Manager and Manager Review

### CODE REVIEW

File: CheckDrive.Api/CheckDrive.Infrastructure/Persistence/Configurations/ManagerReviewConfigurations.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+﻿using CheckDrive.Domain.Entities;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Microsoft.EntityFrameworkCore;
+
+namespace CheckDrive.Infrastructure.Persistence.Configurations;
+
+internal class ManagerReviewConfigurations : IEntityTypeConfiguration<ManagerReview>
```
Comment (FirdavsAX) [inline CheckDrive.Api/CheckDrive.Infrastructure/Persistence/Configurations/ManagerReviewConfigurations.cs:7]: this shoud be sealed class

File: CheckDrive.Api/CheckDrive.Infrastructure/Persistence/Configurations/ManagerReviewConfigurations.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+{
+    public void Configure(EntityTypeBuilder<ManagerReview> builder)
+    {
+        builder.ToTable(nameof(ManagerReview));
+        builder.HasKey(mn => mn.Id);
+
+        #region Relationships
+
```
Comment (FirdavsAX) [inline CheckDrive.Api/CheckDrive.Infrastructure/Persistence/Configurations/ManagerReviewConfigurations.cs:15]: Add Configuration with Debt

File: CheckDrive.Api/CheckDrive.Infrastructure/Persistence/Configurations/ManagerReviewConfigurations.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            .HasForeignKey(mn => mn.ManagerId)
+            .OnDelete(DeleteBehavior.NoAction)
+            .IsRequired();
+
+        #endregion
+
+        #region Properties
+
+        builder
+            .Property(mn => mn.DebtAmountAdjusment)
```
Comment (FirdavsAX) [inline CheckDrive.Api/CheckDrive.Infrastructure/Persistence/Configurations/ManagerReviewConfigurations.cs:35]: rename mn to mr

**Review by Ramadan1011 — APPROVED** [review]
looks good

**Comment by FirdavsAX** [comment]
> looks good

Thanks

**Review by Boburm1rzo — APPROVED** [review]
nice

**Review by Mirazyzz — APPROVED** [review]
LGTM

**Pull request merged**